internal/provider/resource_docker_container_funcs.go

```diff
@@ -560,7 +559,8 @@ func resourceDockerContainerCreate(ctx context.Context, d *schema.ResourceData,
 }
 
 func resourceDockerContainerRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	log.Printf("[INFO] Waiting for container: '%s' to run: max '%v seconds'", d.Id(), containerReadRefreshTimeout)
+	containerReadRefreshTimeoutMilliseconds := d.Get("container_read_refresh_timeout_milliseconds").(int)
+	log.Printf("[INFO] Waiting for container: '%s' to run: max '%v seconds'", d.Id(), containerReadRefreshTimeoutMilliseconds)
```

Comment:
```suggestion
	log.Printf("[INFO] Waiting for container: '%s' to run: max '%v seconds'", d.Id(), containerReadRefreshTimeoutMilliseconds/1000)
```

I think it makes sense to show the correct seconds, instead of milliseconds. My suggestion is probably not the best one, but you get the idea

Reply:
Thanks. I changed this over to milliseconds last minute, and missed this fmt "seconds" left behind. I will update.

Reply:
Fixed this, and also ran the website generation. Still trying to figure out why those tests are failing (they timeout waiting for status running, so my configuration is wrong somewhere, as it should be reading the default)